src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -312,6 +312,9 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
 
     with self.argument_context('acr check-health') as c:
         c.argument('ignore_errors', options_list=['--ignore-errors'], help='Provide all health checks, even if errors are found', action='store_true', required=False)
+        c.argument('vnet', options_list=['--vnet'],
+                   help="Virtual network ID to verify the DNS routing of private endpoints. Can be a name if in the same resource group of the registry. You must run the command inside the virtual network",
```

Comment:
@toddysm, please check out the description and error messages below and see whether it makes sense and friendly to users

Reply:
@yugangw-msft Correct me if I'm wrong - please double-check the usage of VNET throughout the code. Users are expected to provide the Subnet Resource ID or (Subnet name and Vnet name). If they provide the VNET ID, the code may not work since subnetID != VnetID.

Vnet Resource ID: `/subscriptions/{subID}/resourceGroups/{resourceGroup}/providers/Microsoft.Network/virtualNetworks/{vnetName}`

Subnet Resource ID: 
`/subscriptions/{subId}/resourceGroups/adupadhyRG/providers/Microsoft.Network/virtualNetworks/{vnetName}/subnets/{subnetName}`

Usually, subnet name is `default`.

Reply:
For texts, maybe we can use `az network private-endpoint create -h` for inspiration.

![image](https://user-images.githubusercontent.com/20955088/115630523-01eeaa80-a2b9-11eb-9580-d0e4b6e155a0.png)

Reply:
`--subnet`  is more accurate, which I can follow your suggestion.
The context of only matching `vnet` is having 2+ PE in the same vnet (even different subnet) is an anti-pattern we want to capture, as dns routing might become unpredictable.
So, it is your call. Let me know we go with vnet or subnet

Reply:
Makes sense, let's go with VNET.